screens/SCWTest.tsx

```diff
@@ -0,0 +1,165 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { thirdwebClient, thirdwebWallets } from "@utils/thirdweb";
+import React, { useCallback } from "react";
+import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+import { prepareTransaction } from "thirdweb";
+import { ethers5Adapter } from "thirdweb/adapters/ethers5";
+import { base } from "thirdweb/chains";
+import {
+  useActiveAccount,
+  useActiveWallet,
+  useConnect,
+  useDisconnect,
+  useSendTransaction,
+  useSetActiveWallet,
+  useSwitchActiveWalletChain,
+} from "thirdweb/react";
+
+import config from "../config";
+
+const MainScreen: React.FC = () => {
+  const { connect: thirdwebConnect } = useConnect();
+  const activeAccount = useActiveAccount();
+  const activeWallet = useActiveWallet();
+  const { disconnect: thirdwebDisconnect } = useDisconnect();
+  const sendTransaction = useSendTransaction();
+  const setActiveWallet = useSetActiveWallet();
+  const switchChain = useSwitchActiveWalletChain();
+
+  const disconnectWithoutLogin = useCallback(async () => {
+    const storageKeys = await AsyncStorage.getAllKeys();
+    const wcKeys = storageKeys.filter((k) =>
+      k.startsWith("-Coinbase Smart Wallet:")
+    );
+    await AsyncStorage.multiRemove(wcKeys);
+
+    console.log("DONE!");
+  }, []);
+
+  const handleButtonPress = useCallback(() => {
+    thirdwebConnect(async () => {
+      // instantiate wallet
+      const coinbaseWallet = thirdwebWallets["Coinbase Smart Wallet"];
```

Comment:
is thirdwebWallets typed via proper keys or just strings? let's be as strict as we can and use keys if not already doing so

Reply:
this file has been removed since

Reply:
but thirdwebWallets is typed:
`const thirdwebWallets: Record<ISupportedWalletName, Wallet>`